dronegen/tag.go

```diff
@@ -175,7 +176,11 @@ func tagPipelines() []pipeline {
 
 			// RPM/DEB package builds
 			for _, packageType := range []string{rpmPackage, debPackage} {
-				ps = append(ps, tagPackagePipeline(packageType, buildType{os: "linux", arch: arch, fips: fips}))
+				bt := buildType{os: "linux", arch: arch, fips: fips}
```

Comment:
```suggestion
				bt := buildType{os: "linux", arch: arch, fips: fips, centos7: packageType == "rpm"}
```

1. You can inline this without the need for a conditional.
2. Do we need to check arch? If package type is RPM I think it's safe to assume we should use the CentOS builds.